From patchwork Sat Nov  7 21:37:36 2009
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [01/75] netx: declare MODULE_FIRMWARE
Date: Sat, 07 Nov 2009 11:37:36 -0000
From: Ben Hutchings <ben@decadent.org.uk>
X-Patchwork-Id: 37920
Message-Id: <1257629856.15927.370.camel@localhost>
To: David Miller <davem@davemloft.net>
Cc: Sascha Hauer <s.hauer@pengutronix.de>,
	netdev <netdev@vger.kernel.org>

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
Acked-by: Sascha Hauer <s.hauer@pengutronix.de>

---
Note that the filenames are generated in xc_request_firmware(), defined
in arch/arm/mach-netx/xc.c.  I'm not sure whether it's actually worth
doing this for a platform driver.

Ben.

 drivers/net/netx-eth.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/drivers/net/netx-eth.c b/drivers/net/netx-eth.c
index 9f42354..a0d65f5 100644
--- a/drivers/net/netx-eth.c
+++ b/drivers/net/netx-eth.c
@@ -510,3 +510,6 @@ module_exit(netx_eth_cleanup);
 MODULE_AUTHOR("Sascha Hauer, Pengutronix");
 MODULE_LICENSE("GPL");
 MODULE_ALIAS("platform:" CARDNAME);
+MODULE_FIRMWARE("xc0.bin");
+MODULE_FIRMWARE("xc1.bin");
+MODULE_FIRMWARE("xc2.bin");
